broker-core/src/main/java/io/zeebe/broker/logstreams/state/StatePositionSupplier.java

```diff
@@ -70,37 +63,19 @@ public long getLastProcessedPosition() {
       }
     }
 
-    return -1;
+    return new Tuple(exportedPosition, processedPosition);
```

Comment:
Method is called `getProcessedAndExportedPositions` but it returns a tuple with as first value the exported position, this is confusing :sweat_smile: Maybe just rename it as `getLatestPositions`

Reply:
Good point :). Corrected.